### PR TITLE
William/fix action types

### PR DESCRIPTION
**Pull request by swansontec**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [x] n/a

### CODE REVIEW

File: src/modules/Action.js [hunk 1]
```diff
@@ -24,7 +25,9 @@ type ActionType =
   // | 'OPEN_VIEWXPUB_WALLET_MODAL'
   // | 'CLOSE_VIEWXPUB_WALLET_MODAL'
   // | 'UNLOCK_WALLET_SEED'
-  | 'SWAP_FROM_TO_CRYPTO_WALLETS'
+  // | 'SELECT_FROM_WALLET_CRYPTO_EXCHANGE'
+  // | 'SELECT_TO_WALLET_CRYPTO_EXCHANGE'
+  // | 'SWAP_FROM_TO_CRYPTO_WALLETS'
```
Comment (jamesrisberg) [inline src/modules/Action.js]: Should we leave these commented lines here? Or can they be removed?

Comment (jamesrisberg) [inline src/modules/Action.js]: Should we leave these commented lines here? Or can they be removed?

Comment (swansontec) [reply src/modules/Action.js]: When Kevin initially did this work, he had comments for all the properly-typed actions. I don't think this is necessary, but I didn't want to rock the boat, so I left them in.

**Comment by kylanhurt** [comment]
@swansontec You able to fix the merge conflicts for this?

**Comment by swansontec** [comment]
Fixed merge conflicts.

File: src/modules/Action.js (hunk 1 above)
Comment (swansontec) [reply src/modules/Action.js]: I went ahead and removed them, per your comment.

**Pull request merged**